scipy/stats/_continuous_distns.py

```diff
@@ -4346,8 +4346,19 @@ def _stats(self, a, moments='mvsk'):
         return m1, m2, g1, g2
 
     def _entropy(self, a):
-        return a - (a+1.0) * sc.psi(a) + sc.gammaln(a)
+        a = np.asarray([a])
+        def regular(a):
+            h = a - (a+1.0) * sc.psi(a) + sc.gammaln(a)
+            return h
```

Comment:
```suggestion
            h = a - (a + 1.0) * sc.psi(a) + sc.gammaln(a)
            return h
```

As in other places, a problem with the current linting infrastructure.